sonarqube/tests/test_unit_components.py

```diff
@@ -0,0 +1,382 @@
+import re
+from random import randrange
+
+import mock
+import pytest
+
+from .common import HOST, METRICS, PORT
+
+pytestmark = [pytest.mark.unit]
+
+
+@mock.patch("datadog_checks.sonarqube.check.SonarqubeAPI")
+def test_service_check_ok(mock_api, aggregator, dd_run_check, sonarqube_check):
+    # Given
+    instance = {
+        'web_endpoint': 'http://{}:{}'.format(HOST, PORT),
+        'components': {},
+    }
+    check = sonarqube_check(instance)
+    # When
+    dd_run_check(check)
+    # Then
```

Comment:
We should remove these and other comments before merging

Reply:
Same as previous comment